workflows/kube-proxy-all/workflow.yaml

```diff
@@ -13,6 +15,12 @@ spec:
     parameters:
       - name: adminModeNamespace
         value: litmus
+      - name: appNamespace
+        value: "kube-system" 
+      - name: contextCPUHog
+        value: "nodeCpuHog_chaos"
```

Comment:
Can we convert this to `node-cpu-hog_chaos` or `node-cpu-hog_infra`? -- change for all infra experiments